### ISSUE TITLE
Update template when push data (Querying Data)

### ISSUE THREAD

**Issue by absemetov** [firebase/emberfire#263]
I use this model:
```javascript
this.store.find('cart', {
      orderBy: 'timestamp',
      limitToLast: 5
});
```
My template dont show data when event push, but I show data in Ember inspector

But if use 
```javascript
this.store.find('cart');
```
all work correct.

**Comment by tstirrat** [comment]
This is known issue (#235) and there is a fix coming in PR #260 

If you need this functionality please use the branch as explained in [this comment](https://github.com/firebase/emberfire/issues/138#issuecomment-84148825)

```json
"devDependencies": {
  "emberfire": "firebase/emberfire#ts-find-query-with-updates"
```

**Comment by absemetov** [comment]
@tstirrat thanks
